pkg/util/template/func.go

```diff
@@ -117,3 +119,14 @@ func makeInclude(t tpl) func(tplName string, data any) (template.HTML, error) {
 		return html, err
 	}
 }
+
+func trimSpace(input interface{}) string {
```

Comment:
```golang
func trim[T ~string](input T) T {
  return strings.Trim(string(input))
}
```

Reply:
Seems still need type assertion if want to add to template func map. Add comment for this trimHTML func to better explain instead.